### ISSUE TITLE
SearchBox inputprops fontSize

### ISSUE THREAD

**Issue by rkalghatgi** [elastic/search-ui#549]
Hello,

Does SearchBox inputprops and buttonprops accept fontSize or any font input prop? I looked through the source and wasn't sure: https://github.com/elastic/search-ui/blob/master/packages/react-search-ui-views/src/SearchBox.js 

Thanks!

**Comment by rkalghatgi** [comment]
Thanks. I tested that out and it did not work. I don't think fontSizes are supported.

**Comment by JasonStoltz** [comment]
Or sorry, here, rather: https://codesandbox.io/s/national-parks-example-kdyms

**Comment by rkalghatgi** [comment]
Thank you. That did work for SearchBox. Sorry I was testing it out on Facets because I am trying to increase the visibility of the facets. Does it work the same way for Facets? Also, how would I be able to modify the SearchBox button? Would it be buttonProps = {{style: {fontSize: '3rem'}}}?

**Comment by JasonStoltz** [comment]
Yes, that is likely how it works for the SearchBox button, I'd have to look at the docs to know the exact syntax.

You could also do any of these things with an external stylesheet that just targets these elements.

I don't know about Facets. You may need to play around a bit. If it doesn't work with a prop, then you'll need either an external stylesheet or a completely custom view. Both topics are covered in the Advanced.md guide.
